pxd_compat.h

```diff
@@ -62,7 +62,7 @@
 #define BIOSET_CREATE(sz, pad, opt)   bioset_create(sz, pad)
 #endif
 
-#if defined(bio_set_dev)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,14,0) || defined __EL8__
```

Comment:
This means all EL8 are supposed to have bio_set_dev. Is that true? Can we check it quick.
And I would run a few smoke/bvts with a few rhel/ubuntu kernels to sanity check.

Reply:
__EL8__ is defined for RHEL 8/9. From internet sources, I could see that all these distros come with minimum kernel version 4.18.0-80 (https://access.redhat.com/articles/3078) which should have bio_set_dev. 

Will do some smokes runs with few RHEL/Ubuntu kernels